pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-hangouts/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchhg, pidgin, glib, json_glib, protobuf, protobufc }:
+
+stdenv.mkDerivation rec {
+  name = "purple-hangouts-${version}";
+  version = "hg-2016-07-17";
```

Comment:
Along with having teh version only being `2016-07-17`, put the `hg-` prefix in to the name.